backend/src/Server.hs

```diff
@@ -44,9 +44,9 @@ main = do
           hm $
             graphQLApp
               (prometheus p $ libraryServer conn)
-              (Proxy @('Just "Query"))
-              (Proxy @('Just "Mutation"))
-              (Proxy @('Just "Subscription"))
+              (Proxy @( 'Just "Query"))
+              (Proxy @( 'Just "Mutation"))
+              (Proxy @( 'Just "Subscription"))
```

Comment:
I'm afraid the latest version of ormolu requires you to leave it as before 😉

Reply:
I don't have globally installed ormolu

Reply:
`nix-env -i ormolu` 😉

Reply:
We need an ormolu config, not a "latest version".